bin/cli.js

```diff
@@ -48,11 +48,17 @@ async function initKnex(env, opts) {
     ? await openKnexfile(env.configPath)
     : mkConfigObj(opts);
 
-  const resolvedConfig = resolveEnvironmentConfig(
-    opts,
-    env.configuration,
-    env.configPath
-  );
+  const resolvedConfig = resolveEnvironmentConfig(opts, env.configuration);
+
+  // override connection, client and migrationsTableName
+  // if specified through cmd line arguments
+
+  if (opts.connection) resolvedConfig.connection = opts.connection;
```

Comment:
How do you feel about resolvedConfig.x = opts.x || resolvedConfig style?

Reply:
I can fix that !!

Reply:
Done in 0780b70